data/bingo/functions/tick/tick.mcfunction

```diff
@@ -18,19 +18,31 @@ execute if score $is_multiplayer bingo.state matches 2.. run scoreboard players
+# Send Resource pack check chat message
+#
+# Uses technical translation strings with unused argument indexes
+# (1, 2 are used indexes)
+# - "%1074992263$s%1$s" - 'bingo' encoded* as 1074992263; %1$s - 1st argument
+# - "%1074992263$s%<X>$s%1$s" - as above; version-check iteration as <X> (3+)
+# - both above translate to "%2$s" (2nd argument), in [en_us] lang-file
+# 
+# *NeunEinser's encoder: https://github.com/NeunEinser/translation-placeholder-namespace
+#tellraw @a[scores={bingo.resources=1}] ["\n\n\n\n\n", {"translate": "%1074992263$s%1$s", "with": [["",{"text": "The resourcepack is currently not enabled.", "color": "red"}, "\nIf there is a server resource pack in place, enable it and wait until the Mojang loading screen is done. Then click “re-check” down below.\n\nOtherwise, please download the Resourcepack ", {"text": "here", "color": "#00c3ff", "clickEvent": {"action": "open_url", "value": "https://github.com/NeunEinser/bingo/releases/download/5.0-pre1/Resourcepack.zip"}}, " and enable it.\n\n[", {"text": "Re-check", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.resources"}}, "]"], ["", {"translate": "%1074992263$s%3$s%1$s", "with": [["", {"text": "Wrong resourcepack version is loaded.", "color": "red"}, "\nYou are using a version of the resource pack for other version of Bingo.\n\nPlease redownload the correct Resourcepack for this Bingo version ", {"text": "here", "color": "#00c3ff", "clickEvent": {"action": "open_url", "value": "https://github.com/NeunEinser/bingo/releases/download/5.0-pre1/Resourcepack.zip"}}, ", enable it and click “re-check” below.\n\n[", {"text": "Re-check", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.resources"}}, "]"], ["", {"translate": "bingo.resourcepack_check.go_to_lobby.part1", "color": "green"}, "\n", {"translate": "bingo.resourcepack_check.go_to_lobby.part2", "with": [{"translate": "bingo.resourcepack_check.go_to_lobby.click", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.resources set 91"}}]}]]}]]}]
```

Comment:
Currently `rp-version-check` text says (say if it needs changing):
```
{Wrong resourcepack version is loaded*},
You are using a version of the resource pack for other version of Bingo.

Please redownload the correct Resourcepack for this Bingo version {here**}, enable it and click “re-check” below.

[{Re-check***}]
```
\* - color: red
\** - color: #00c3ff; url: https://github.com/NeunEinser/bingo/releases/download/5.0-pre1/Resourcepack.zip
\*** - color: #00c3ff

Reply:
I'd suggest
```
[red]Incorrect resource pack version enabled[/red]
You are using an incompatible version of the Bingo resource pack.

Please download the correct version [link]here[/link], enable it and click “re-check” down below.
[clickEvent]Re-check[/clickEvent]
```

Reply:
Perhaps also something like "If the resource pack was applied automatically through a server resource pack, please inform the server admin to update the server resource pack.

But maybe it gets to lengthy then. It's always a tradeoff between being too spammy and informative.

Reply:
Maybe send it on condition if `is_multiplayer`?

Reply:
Hmm, if we add a version number to is_multiplayer as well, that won't work though.

Reply:
Nevermind the _multiplayer condition_ thing - it would add another place to update the version check string.

I think adding the info about server-rp should be added, and IMO it won't really be spammy - it shows only during the check, when the rp-version is wrong (and we can assume someone will have the correct one in the next check already)

Reply:
Text change (with server part) done in https://github.com/NeunEinser/bingo/pull/111/commits/d584a2bba7a0923b8546863d161046997b3a6c96